### PR TITLE
Feature1 create student

### PR DESCRIPTION
**Pull request by hassa963**
I have written tests for all the entities it can most definitely be more thorough but its a start

### CODE REVIEW

**Review by jpmedinagl — APPROVED** [review]
These are a good set of tests for now ! We'll write more in the future as well !

File: build.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ repositories {
 dependencies {
     implementation 'junit:junit:4.13.1'
     testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
+    testImplementation 'org.testng:testng:7.1.0'
```
Comment (jpmedinagl) [inline build.gradle:12]: I'm wondering what did this line of code change? Was it required in the testing?

**Review by jpmedinagl — COMMENTED** [review]
I'm not requesting changes, however I am wondering why did we change the gradle file? Did something not work if that line wasn't there?

Could you explain it before I approve it?

**Review by jpmedinagl — CHANGES_REQUESTED** [review]
Just requesting changes for the explanation

File: build.gradle (hunk 1 above)
Comment (hassa963) [reply build.gradle:12]: It kind of just added it and refuse to work with out it so I'm not sure why it did that

**Comment by hassa963** [comment]
It kind of just added it and refused to work without it so I'm not sure why it did that but for whatever reason some tests just wouldn't run so I followed the context action it provided

**Review by jpmedinagl — APPROVED** [review]

**Comment by jpmedinagl** [comment]
Okay that makes sense ! Thank you for letting us know why you added that.

**Pull request merged**